iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings

```diff
@@ -811,7 +811,7 @@
 
 "placepage_delete_bookmark_button" = "Könyvjelző törlése";
 
-"editor_edits_sent_message" = "Javasolt változtatásainak elküldése sikerült";
+"editor_edits_sent_message" = "Az Ön megjegyzése elküldésre kerül az OpenStreetMap-nek";
```

Comment:
Hm, DeepL says that `Az Ön változtatási javaslata elküldésre került` means `Your change proposal has been sent`.

But we want to emphasize that change is not sent yet, it will be sent at some time in the future.

Reply:
@d4f5409d happy new year! Waiting for your response here.

Reply:
HNY! Don't rely on DeepL that much, it isn't 100% perfect ;)
I changed the translation accordingly to your review.

Reply:
updated ✔️